backend/routes.js

```diff
@@ -219,18 +239,18 @@ route.POST('/event', {
   }
 })
 
-route.GET('/eventsAfter/{contractID}/{since}/{limit?}', {}, async function (request, h) {
+route.GET('/eventsAfter/{contractID}/{since}/{limit?}', {
+  validate: {
+    params: Joi.object({
+      contractID: Joi.string().regex(CID_REGEX).required(),
+      since: Joi.string().regex(POSITIVE_INTEGER_REGEX).required(),
+      limit: Joi.string().regex(POSITIVE_INTEGER_REGEX)
```

Comment:
`limit` is marked as being a string but `'backend/db/streamEntriesAfter'` expects a `number`

Reply:
I'd think that we have no choice here, but it can be coerced into a number.